packages/gestalt/src/SheetMobile/PrimaryAction.js

```diff
@@ -0,0 +1,50 @@
+// @flow strict
+import { type Node, type ElementConfig } from 'react';
+import Button from '../Button.js';
+import Link from '../Link.js';
+
+type Props = {|
+  accessibilityLabel: string,
+  href?: string,
+  label: string,
+  onClick?: $ElementType<ElementConfig<typeof Button>, 'onClick'>,
+  rel?: $ElementType<ElementConfig<typeof Link>, 'rel'>,
+  size?: $ElementType<ElementConfig<typeof Button>, 'size'>,
+  target?: $ElementType<ElementConfig<typeof Link>, 'target'>,
+|};
+
+export default function ToastPrimaryAction({
```

Comment:
It looks like this was perhaps copied from somewhere. 😅 Is a separate component needed? Could a PrimaryAction component be shared between the two?

Reply:
DONE